lib/iris/loading.py

```diff
@@ -469,6 +487,32 @@ class LoadProblems(threading.local):
     class Problem:
         """A single object that could not be loaded correctly."""
 
+        # TODO: is it acceptable to use dataclass instead of NamedTuple purely
+        #  because the docs render cleaner?!
```

Comment:
‼

Reply:
namedtuples are immutable, so you wouldn't be able to do things like:

```python
problem = _add_or_capture( .... )
if problem is not None and can_handle_it:
  handle_it()
  problem.handled = True  # immutable if named tuple?
```

So the better rendered docs is just a bonus!?